### ISSUE TITLE
Warnings should include column number

### ISSUE THREAD

**Issue by cgay** [dylan-lang/opendylan#1214]
Given that we already highlight the offending part of a source code line in warnings:
```
make-dylan-app.dylan:84: Serious warning - Reference to undefined binding {xcompile-regex in make-dylan-app}.
                           --------------
          if (regex-search(xcompile-regex("[a-z][a-z0-9-]*"), library-name))
                           --------------
```
it should be easy to add the column number after the line number so that emacs (and others, I assume) can jump us right to the column we probably need to edit.

Pretty sure that means: `make-dylan-app.dylan:84:48: Serious warning ...`

**Comment by housel** [comment]
Line and column ranges are allowed in the error message format as well, see https://www.gnu.org/prep/standards/html_node/Errors.html

**Comment by BarAgent** [comment]
Yep, but for clarity, I’d recommend leaving columns out if you have a line range.

**Comment by housel** [comment]
Fixed in PR #1276
